.circleci/config.yml

```
@@ -231,7 +226,7 @@ jobs:
       - run:
           name: Test arm with no gpu
           command: |
-            cargo +<< pipeline.parameters.nightly-toolchain >> -Zpackage-features test --release --all --verbose --no-default-features
+            cargo +$(cat rust-toolchain) -Zpackage-features test --release --all --verbose --no-default-features
```

```suggestion
            cargo +$(cat rust-toolchain) test --release --all --verbose --no-default-features
```
The `package-features` feature has been stabilized, things should work without it.